### ISSUE TITLE
http status 400 BAD REQUEST / Bridge not reachable after some hour

### ISSUE THREAD

**Issue by Blog404DE** [ebaauw/homebridge-hue#836]
#### Software-Versions
- Raspberry Pi 4 with current Rasbian Buster Version
- Node.js Version v12.20.0
- Npm Version v6.14.8
- Homebridge v1.3.0-beta.27 (testet with v1.2.3 and v1.1.6)
- homebridge-hue v0.12.3
- ... and some other homebridge plugins (most important homebridge-config-ui-x v4.34.0, homebridge-homematic v0.1.231)

#### Configuration

```json
{
	"platform": "Hue",
	"name": "Hue",
	"host": "192.168.0.127",
	"users": { xxxx	},
	"parallelRequests": 3,
	"lights": true,
	"sensors": false,
	"scenes": false,
	"groups": false,
	"rooms": false,
	"nativeHomeKitLights": true,
	"nativeHomeKitSensors": true,
	"linkButton": false
}
```

#### Issue

I've connected my Hue Bridge (v2) with some devices (Alexa, iPhone, Mac and Homematic and Homebridge). After a random time (or seems random / some hours or even 1-2 days) my lights didn't respond over homebridge and homebridge is logging the following error:

```
[2020-11-29 3:00:24] [Hue] Philips hue: bridge request 23482: get /lights
[2020-11-29 3:00:24] [Hue] Philips hue: bridge communication error: get /lights: http status 400 BAD REQUEST
[2020-11-29 3:00:24] [Hue] Philips hue: heartbeat error: get /lights: http status 400 BAD REQUEST
```

After some time, the connection to the bridge is working again. You can see it on my attached logs on the following timestamps: 
[2020-11-29 2:53:59] (still working) -> [2020-11-29 3:00:24] (offline) -> [2020-11-29 9:52:49] (working again)

I've you restart homebridge during the errors you see directly:

```
[2020-11-29 13:03:48] [Hue] 192.***.***.127: 192.168.0.127:80: cannot connect: ECONNREFUSED - retrying in 15s
[2020-11-29 13:04:03] [Hue] 192.***.***.127: 192.168.0.127:80: cannot connect: ECONNREFUSED - retrying in 15s
```

During the time the bridge wasn't responding, i've made some interessting tests:
nmap from my raspberry showed a closed Port 80 on my hue bridge. The same time, nmap from my workstation showed port 80 open on my bridge and other clients (like Homematic) could still control my lights. 
It seems, like the hue bridge is blocking connections from my raspberry after some time. 

#### Log Messages
-> added Log-File below (sorry, I had no debug-log, because it sometimes took hours, somedays 1 or 2 days until this error happen

#### Debug Files
[homebridge-hue.json.gz](https://github.com/ebaauw/homebridge-hue/files/5612148/homebridge-hue.json.gz)
[hue.log.gz](https://github.com/ebaauw/homebridge-hue/files/5612166/hue.log.gz)

**Comment by ebaauw** [comment]
These are issues with the network connection to your Hue bridge, nothing Homebridge Hue can do here.  I can only recommend to double-check your firewall(s) and network setup.

**Comment by Blog404DE** [comment]
thx for your help ... I've really searched nearly 2h and did not think of the easiest failure source. The Raspberry had (currently) no local firewall but I've checked nftable/iptable and found no active rules (as expected). I've disabled pi-hole (just to eliminate it). 

Where did the problem come from? ... I'm running a DHCP Server and nearly every device in my office got a static ip and I've got a small list with every device and his current ip. Just one device was missing and this device got (👐@Murphy's Law) got the same (static) IP as my Hue Bridge and to make it even better - this device had port 443 open. That's the reason why Nmap every time showed port 443 open, but sometimes port 80 closed. 

Small mistake, big impact.
